### DISCUSSION TITLE
[blocks]: Image gallery modal

### DISCUSSION THREAD

**Discussion by Bazyli12** [shadcn-ui/ui#3321] [Blocks Request]
### Description

Image gallery often is used on ecommerce sites to display multiple product images.

It can be created using dialog and carousel component.

example:
![image](https://github.com/shadcn-ui/ui/assets/79319856/1a001af6-08f2-4811-b9c4-42533ee23b07)

### Example

https://www.x-kom.pl/p/1179980-smartfon-telefon-apple-iphone-15-128gb-black.html#modal:galeria

**Comment by sovetski** [comment]
+1 for this!
